en/core-libraries/inflector.rst

```diff
@@ -13,6 +13,57 @@ statically. Example:
 You can try out the inflections online at `inflector.cakephp.org
 <http://inflector.cakephp.org/>`_.
```

Comment:
I'm attaching the page will all the results for your perusal in case you need it.
[Inflector_results.html.zip](https://github.com/cakephp/docs/files/627056/Inflector_results.html.zip)